src/app/core/export/export-service/export.service.spec.ts

```diff
@@ -0,0 +1,138 @@
+import { TestBed } from "@angular/core/testing";
+
+import { ExportService } from "./export.service";
+import { ConfigurableEnumValue } from "../../configurable-enum/configurable-enum.interface";
+import { DatabaseField } from "../../entity/database-field.decorator";
+import { DatabaseEntity } from "../../entity/database-entity.decorator";
+import { Entity } from "../../entity/model/entity";
+import { BackupService } from "../../admin/services/backup.service";
+
+describe("ExportService", () => {
+  let service: ExportService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [ExportService, QueryService],
+    });
+
+    service = TestBed.inject<ExportService>(ExportService);
+  });
+
+  it("should be created", () => {
+    expect(service).toBeTruthy();
+  });
+
+  it("should create the correct json object", () => {
+    class TestClass {
+      propertyOne;
+      propertyTwo;
+    }
+    const test = new TestClass();
+    test.propertyOne = "Hello";
+    test.propertyTwo = "World";
+
+    const result = service.createJson([test]);
+
+    const expected = JSON.stringify({
+      propertyOne: "Hello",
+      propertyTwo: "World",
+    });
+    expect(result).toEqual(expected);
+  });
+
+  it("should transform a json array", () => {
+    class TestClass {
+      propertyOne;
+      propertyTwo;
+    }
+    const test = new TestClass();
+    test.propertyOne = "Hello";
+    test.propertyTwo = "World";
+
+    const result = service.createJson([test, test]);
+
+    let expected = JSON.stringify({
+      propertyOne: "Hello",
+      propertyTwo: "World",
+    });
+    expected += BackupService.SEPARATOR_ROW;
```

Comment:
Why is the `SEPARATOR_ROW` of the backup service and not the export service used